tests/30rooms/60version_upgrade.pl

```diff
@@ -501,21 +504,6 @@ sub upgrade_room_synced {
       }
 
       $f->then( sub {
-         # to make things harder, we now restrict our ability to change each of
-         # those states: the server should make sure it sets up the state
-         # *before* it replicates the PL.
-         matrix_change_room_power_levels(
-            $creator, $room_id, sub {
-               my ( $levels ) = @_;
-               foreach my $k ( keys %STATE_DICT ) {
-                  $levels->{events}->{$k} = 80;
-               }
-               $levels->{users}->{$creator->user_id} = 50;
-            },
-         );
-      })->then( sub {
-         matrix_sync( $creator );
```

Comment:
Why are we removing this sync?

Reply:
I presumed it was required to sync the power level changes, but perhaps it's there for some other reason?

(In general I'm not often sure why a `matrix_sync` is done except in only obvious scenarios).

Reply:
Its so that `matrix_sync_again` only shows changes from that point

Reply:
`upgrade_room_synced` syncs again though? But perhaps with an ephemeral copy of the sync token?

Reply:
the `_synced` stuff *should* reset the sync token to what it was before

Reply:
Huh! Ok, good to know. I'll add it back.